src/goes/goes_converter.py

```diff
@@ -308,18 +448,19 @@ def _create_preqc_brightness_temperature_variable(self):
         data_array = temp_dict[0]
         for i in range(1, counter):
             data_array = np.column_stack((data_array, temp_dict[i]))
-        self._output_dataset_bt.createVariable('/PreQC/brightness_temperature', 'f4', ('nlocs', 'nchans'),
-                                               fill_value=-999)
-        self._output_dataset_bt['/PreQC/brightness_temperature'][:] = data_array
-        self._output_dataset_bt['/PreQC/brightness_temperature'].setncattr('flag_values', '0,1,2,3')
-        self._output_dataset_bt['/PreQC/brightness_temperature'].setncattr('flag_meanings',
-                                                                           'good_pixel_qf '
-                                                                           'conditionally_usable_pixel_qf '
-                                                                           'out_of_range_pixel_qf no_value_pixel_qf')
+        output_dataset.createVariable('/PreQC/brightness_temperature', 'f4', ('nlocs', 'nchans'),
+                                      fill_value=-999)
+        output_dataset['/PreQC/brightness_temperature'][:] = data_array
+        output_dataset['/PreQC/brightness_temperature'].setncattr('flag_values', '0,1,2,3')
+        output_dataset['/PreQC/brightness_temperature'].setncattr('flag_meanings',
+                                                                  'good_pixel_qf '
+                                                                  'conditionally_usable_pixel_qf '
+                                                                  'out_of_range_pixel_qf no_value_pixel_qf')
```

Comment:
Is this flag meanings section done correctly?  If I was to guess, I would expect flag value 0 to mean 'good_pixel', flag value 1 to mean 'conditionally_usable_pixel', flag value 2 to mean 'out_of_range_pixel' and flag value 3 to mean 'no_value_pixel.'  The way it's written (here plus a duplicate in next method) appears to have a space between the out-of-range and no-value so I would be expecting the list to be 4 items for flag_meanings.

Reply:
This has been resolved in the latest commit.